test/core.jl

```diff
@@ -35,4 +35,4 @@ prog_perf(10^7)
 noprog_perf(10^7)
 @time prog_perf(10^7)
 @time noprog_perf(10^7)
-@test @elapsed(prog_perf(10^7)) < 40*@elapsed(noprog_perf(10^7))
+@test @elapsed(prog_perf(10^7)) < 8*@elapsed(noprog_perf(10^7))
```

Comment:
Based on local tests this could go down to 5, but CI seems closer to 6/7. 8 to be conservative